autogluon/task/image_classification/image_classification.py

```diff
@@ -153,24 +152,27 @@ def fit(dataset,
         ngpus_per_trial : int
             How many GPUs to use in each trial (ie. single training run of a model).
         output_directory : str
-            Dir to save search results.
+            Checkpoints of the search state are written to
+            os.path.join(output_directory, 'exp1.ag')
         search_strategy : str
             Which hyperparameter search algorithm to use.
             Options include: 'random' (random search), 'skopt' (SKopt Bayesian optimization), 'grid' (grid search), 'hyperband' (Hyperband), 'rl' (reinforcement learner)
         search_options : dict
             Auxiliary keyword arguments to pass to the searcher that performs hyperparameter optimization.
         resume : bool
-            If a model checkpoint file exists, model training will resume from there when specified.
+            If True, the hyperparameter search is started from state loaded from
+            os.path.join(output_directory, 'exp1.ag')
         dist_ip_addrs : list
             List of IP addresses corresponding to remote workers, in order to leverage distributed computation.
+        scheduler_options : dict
+            Extra arguments passed to __init__ of scheduler, to configure the
+            orchestration of training jobs during hyperparameter-tuning.
         verbose : bool
             Whether or not to print out intermediate information during training.
         plot_results : bool
             Whether or not to generate plots summarizing training process.
         visualizer : str
             Describes method to visualize training progress during `fit()`. Options: ['mxboard', 'tensorboard', 'none'].
-        grace_period : int
```

Comment:
why are we removing this option?

Reply:
Ask by @jwmueller . Now, all scheduler-specific options are passed in scheduler_options.

Reply:
Consider moving as a kwargs option with deprecation warning that it will be removed in future versions. If present in kwargs, override value in scheduler_options for grace_period. This avoids a breaking change. See 'enable_fit_continuation' argument in tabular as an example.

Reply:
OK, good idea!